setup.py

```diff
@@ -43,7 +43,7 @@
     "Mathics-Scanner >= 1.3.0.dev0",
     # Pillow 9.1.0 supports BigTIFF with big-endian byte order.
     # ExampleData image hedy.tif is in this format.
-    "pillow >= 9.1.0",
+    "pillow == 9.2.0",
```

Comment:
Why is this not ">= 9.2.0" ?

Reply:
This change do not belong here. It stay here by accident.